### PR TITLE
Add "How Much?" estimator.

### PR DESCRIPTION
**Pull request by tannerwernecke**
Users might want to know how much they can raise using our software
before actually committing to using it. We want to give them an
opportunity to see how much they can raise before committing to use our
software.

### CODE REVIEW

File: app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -1,2 +1,55 @@
 <% provide(:h1, "About") %>
-<p>This is the about page for public school's walk-a-thon fundraiser</p>
\ No newline at end of file
+<h3>How much could you raise?</h3>
```
Comment (mattzollinhofer) [inline app/views/static_pages/about.html.erb]: accessibility would ask us to use semantic heading tags

```suggestion
<h2>How much could you raise?</h2>
```

File: app/views/static_pages/about.html.erb [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+
+Number of students:<br>
+<input id="numStudents"><br>
+Percent of participation:<br>
+<input id="participation" type="range" min="50" max="100" value="75"><p id="sl1"></p><br>
+Number of donors per student:<br>
+<input id="donors" type="range" min="1" max="25" value="5"><p id="sl2"></p>
+<button onclick="calculate()">Calculate</button>
+<h4 id="answer"></h4>
```
Comment (mattzollinhofer) [inline app/views/static_pages/about.html.erb]: same here a11y
```suggestion
<h3 id="answer"></h3>
```

File: app/views/static_pages/about.html.erb [hunk 3]
```diff
@@ -1,2 +1,55 @@
 <% provide(:h1, "About") %>
-<p>This is the about page for public school's walk-a-thon fundraiser</p>
\ No newline at end of file
+<h3>How much could you raise?</h3>
+
+Number of students:<br>
+<input id="numStudents"><br>
+Percent of participation:<br>
+<input id="participation" type="range" min="50" max="100" value="75"><p id="sl1"></p><br>
```
Comment (mattzollinhofer) [inline app/views/static_pages/about.html.erb]: let's try to find a more understandable `id` name. 

Maybe `slider-1`?

File: app/views/static_pages/about.html.erb [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+Number of donors per student:<br>
+<input id="donors" type="range" min="1" max="25" value="5"><p id="sl2"></p>
```
Comment (mattzollinhofer) [inline app/views/static_pages/about.html.erb]: same here? `slider-2` or `donor-slider`

File: app/views/static_pages/about.html.erb [hunk 5]
```diff
@@ -1,2 +1,55 @@
...
+<button onclick="calculate()">Calculate</button>
+<h4 id="answer"></h4>
+
+<script>
+  //Update slider for participation
+  var slider1 = document.getElementById("participation");
+  document.getElementById("sl1").innerHTML = slider1.value;
+
+  slider1.oninput = function() {
+    document.getElementById("sl1").innerHTML = this.value;
```
Comment (mattzollinhofer) [inline app/views/static_pages/about.html.erb]: don't forget to change this name to match

**Comment by mattzollinhofer** [comment]
addresses #7

File: app/views/static_pages/about.html.erb (hunk 3 above)
Comment (tannerwernecke) [reply app/views/static_pages/about.html.erb]: I realized this doesn't even have to do with a slider. This id is for the amount of participation. I thought `participation-amount` would make more sense here.

File: app/views/static_pages/about.html.erb (hunk 4 above)
Comment (tannerwernecke) [reply app/views/static_pages/about.html.erb]: Same here, I realized `donor-amount` makes more sense.

**Pull request merged**